### ISSUE TITLE
The view was show as decimal on array data

### ISSUE THREAD

**Issue by muhammad-ihsan** [himelbrand/react-native-numeric-input#18]
Well hello, @himelbrand  thanks for this awesome plugins ❤️ 

i tried to create input stepper button on my cart like:

```
import React, { Component } from 'react';
import ....;

export default class Cart extends Component {
  state = {
    textInputs: [],
  };

  render() {

    console.log(this.state.textInput);

    return (
      {cart.get.map((item, index) => (
	<ListItem key={index} noIndent>
	   <Body>
	     <Text>{item.product.name}</Text>
	     <Text note numberOfLines={1}>{item.qty} x {item.price}</Text>
	     <Text note numberOfLines={1}>Rp. {item.total}</Text>
	  </Body>
          <Right>
              <NumericInput initValue={item.qty} value={this.state.textInputs[index] ? this.state.textInputs[index] : item.qty} totalHeight={30} onChange={value => { let { textInputs } = this.state; textInputs[index] = value; this.setState({textInputs, })}}/>
            </Right>
         </ListItem>
      ))}
    );
  }
}
```

the value show as a decimal `12.000000000` in the view, but the console say it is `12`.
I was looking for the solution with this [#13](#13 ) issue.

And all working fine if i delete this:

1. [ComponentWillReceiveProps](https://github.com/himelbrand/react-native-numeric-input/blob/master/NumericInput/NumericInput.js#L21)
2. .toFixed(12) on the [Line 36](https://github.com/himelbrand/react-native-numeric-input/blob/master/NumericInput/NumericInput.js#L36) and  [Line 45](https://github.com/himelbrand/react-native-numeric-input/blob/master/NumericInput/NumericInput.js#L45).

react-native-numeric-input version 1.6.2

**Comment by himelbrand** [comment]
@muhammad-ihsan , I think I found the problem, I'll try and fix it today.
thank you for letting me know

**Comment by himelbrand** [comment]
closed, fixed in the latest version: 1.6.3
if still, this problem occurs, open a new issue.
